ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -564,6 +564,37 @@ tests:
           cpu: 100m
           memory: 200Mi
       timeout: 6h0m0s
+- as: osde2e-stage-aws-addon-mcg-osd
+  cron: 0 6,18 * * *
+  steps:
+    test:
+    - as: addon-mcg-osd-tests
+      commands: |
+        export ADDON_IDS="mcg-osd"
+        export ADDON_POLLING_TIMEOUT="14400"
+        export ADDON_TEST_HARNESSES="quay.io/osd-addons/mcg-osd:harness"
+        export CHANNEL="alpha"
+        export CONFIGS="aws,stage,e2e-suite,addon-suite"
```

Comment:
@jeremyeder can you confirm if `e2e-suite` is required provided we're running `addon-suite` ?
For `ocs-consumer` addon we ended up running only `addon-suite`:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml#L535

Reply:
I included this to run operator tests as well.

https://github.com/openshift/osde2e/blob/main/configs/addon-suite.yaml
https://github.com/openshift/osde2e/blob/main/configs/e2e-suite.yaml